packages/mongo-nav/src/types.ts

```diff
@@ -1,13 +1,23 @@
-const OnElementClick = {
+const NavElement = {
```

Comment:
Much happier with this combined, and also makes the OnElementClick logic more readable 👍